<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>movedoc</class>
 <widget class="QDialog" name="movedoc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget_commodity_movement" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>441</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_widget_commodity_movement">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Рух товарів</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Звідки/Куди</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>361</width>
      <height>151</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed</set>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Назва </string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Кільскість</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>None</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit_date">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>2017-04-11</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Дата</string>
    </property>
   </widget>
   <widget class="QLabel" name="ErrorLabel_Order">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="locale">
     <locale language="Ukrainian" country="Ukraine"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>220</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>380</y>
      <width>168</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_ok">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Закрити</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2_add_next_commodity">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ок</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="comboBox_direction">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Касса-&gt;Склад   </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Склад-&gt;Касса   </string>
     </property>
    </item>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>label_widget_commodity_movement</zorder>
   <zorder>label_21</zorder>
   <zorder>tableWidget</zorder>
   <zorder>lineEdit_date</zorder>
   <zorder>label_16</zorder>
   <zorder>ErrorLabel_Order</zorder>
   <zorder>addButton</zorder>
   <zorder>spinBox</zorder>
   <zorder>comboBox_direction</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
